src/components/FileList.jsx

```diff
@@ -10,12 +10,14 @@ import File from '../components/File'
 
 import SelectionBar from '../containers/SelectionBar'
 import FileActionMenu from '../containers/FileActionMenu'
+import DeleteConfirmation from '../containers/DeleteConfirmation'
 
 const FileList = ({
-  t, f, files, error, showSelection, showActionMenu, onFileEdit, onFolderOpen, onFileToggle, onShowActionMenu
+  t, f, files, error, showSelection, showActionMenu, showDeleteConfirmation, onFileEdit, onFolderOpen, onFileToggle, onShowActionMenu
 }) => (
   <div role='contentinfo'>
     {showSelection && <SelectionBar />}
+    { showDeleteConfirmation && <DeleteConfirmation /> }
```

Comment:
I don't know if there should be 1 or zero space inside braces on L19 and L20. But it is not homogeneous.